src/main/java/com/stripe/model/ShippingDetails.java

```diff
@@ -9,6 +9,8 @@
 @EqualsAndHashCode(callSuper = false)
 public final class ShippingDetails extends StripeObject {
   protected Address address;
+  protected String carrier;
   protected String name;
   protected String phone;
+  protected String trackingNumber;
```

Comment:
This is a bit unrelated but we did not support those properties even though they are valid on `Order`. They are not valid on `Charge`. We likely should revisit `shipping` as a whole in the API though.